pkg/core/resources/manager/cache.go

```diff
@@ -96,7 +99,7 @@ func (c *cachedManager) List(ctx context.Context, list model.ResourceList, fs ..
 		if !found {
 			// After many goroutines are unlocked one by one, only one should execute this branch, the rest should retrieve object from the cache
 			c.metrics.WithLabelValues("list", string(list.GetItemType()), "miss").Inc()
-			if err := c.delegate.List(ctx, list, fs...); err != nil {
+			if err := c.delegate.List(ctx, list, append(fs, store.ListOrdered())...); err != nil {
```

Comment:
I guess this means we'll sort all list requests now, right? I'd go another way and include the `ordered` flag into `cacheKey`. So that if you have 2 consecutive List requests with and without `ListOrdered` you'll get 2 different entries in the cache

Reply:
The question is do we really want unordered data? If we always keep data ordered we might avoid problems with inconsistent results.

Reply:
I don't know, can we be sure it's not a performance hit? 
Also, If we want all data sorted, then probably it makes sense to implement this at the store level. For example, Postgres and Memory can return sorted lists

Reply:
I am not sure about the performance hit, probably that would require some testing. I've changed to keeping the ordered as a key